app/src/main/java/org/mozilla/fenix/search/SearchDialogFragment.kt

```diff
@@ -665,6 +674,50 @@ class SearchDialogFragment : AppCompatDialogFragment(), UserInteractionHandler {
         startActivityForResult(speechIntent, VoiceSearchActivity.SPEECH_REQUEST_CODE)
     }
 
+    private fun addQrButton(searchFragmentState: SearchFragmentState) {
+        if (qrButtonAlreadyAdded) return
+        val searchEngine = searchFragmentState.searchEngineSource.searchEngine
+
+        val isVisible =
```

Comment:
```suggestion
        if (qrButtonAlreadyAdded) return
        
        val searchEngine = searchFragmentState.searchEngineSource.searchEngine
        val isVisible =
```
Just prefer to have a new line after the if statement return so we know that's where the real work begins